telematic_system/telematic_cloud_messaging/src/main/java/com/telematic/telematic_cloud_messaging/nats_influx_connection/InfluxDataWriter.java

```diff
@@ -1,140 +1,140 @@
 
 
-import java.util.concurrent.TimeUnit;
-import java.util.List;
 import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import com.influxdb.client.*;
+import com.influxdb.client.InfluxDBClient;
+import com.influxdb.client.InfluxDBClientFactory;
 import com.influxdb.client.InfluxDBClientOptions;
+import com.influxdb.client.WriteApi;
 import com.influxdb.client.domain.WritePrecision;
+import com.telematic.telematic_cloud_messaging.message_converters.JSON2KeyValuePairsConverter;
 import com.telematic.telematic_cloud_messaging.message_converters.JSONFlattenerHelper;
 
 import okhttp3.OkHttpClient;
 
-import com.telematic.telematic_cloud_messaging.message_converters.JSON2KeyValuePairsConverter;
-
-import org.json.*;  
-import org.slf4j.LoggerFactory;
-import org.slf4j.Logger;
-import org.apache.commons.lang3.exception.ExceptionUtils;
-
 /**
  * The InfluxDataWriter object creates a client with the InfluxDb and publishes data that has been
  * received from the nats server. Required parameters for connection to InfluxDb are included in
  * the config.properties file.
  */
-public class InfluxDataWriter {
-    
-    Config config_;
-    String influx_bucket;
+public class InfluxDataWriter {    
+    private static final Logger logger = LoggerFactory.getLogger(InfluxDataWriter.class);
 
-    boolean influx_connected;
+    Config influxConfig;
+    String influxBucket;
+    boolean influxConnected;
     InfluxDBClient influxDBClient;
     InfluxDBClientOptions adminClientOptions;
     WriteApi writeApi;
 
-    private static final Logger logger = LoggerFactory.getLogger(InfluxDataWriter.class);
-
     /**
      * Constructor to instantiate InfluxDataWriter object
      */
-    public InfluxDataWriter(Config config, Config.BucketType bucket_type) {
+    public InfluxDataWriter(Config config, Config.BucketType bucketType) {
         logger.debug("Creating new InfluxDataWriter");
 
-        if(bucket_type.equals(Config.BucketType.PLATFORM)){
-            this.influx_bucket = config.influx_bucket_platform;
+        if(bucketType.equals(Config.BucketType.PLATFORM)){
+            this.influxBucket = config.influxBucketPlatform;
         }
-        else if(bucket_type == Config.BucketType.STREETS){
-            this.influx_bucket = config.influx_bucket_streets;
+        else if(bucketType == Config.BucketType.STREETS){
+            this.influxBucket = config.influxBucketStreets;
         }
-        else if(bucket_type == Config.BucketType.CLOUD){
-            this.influx_bucket = config.influx_bucket_cloud;
+        else if(bucketType == Config.BucketType.CLOUD){
+            this.influxBucket = config.influxBucketCloud;
         }
         
-        config_ = config;
-        influx_connected = false;
+        influxConfig = config;
```

Comment:
Do we need a influxConfig object now? Seems like we can directly call config from anywhere

Reply:
I tried, but it seems to be more complicated than it looks. This inffluxDateWriter has some other dependencies, this change requires refactor of inffluxDateWriter and their dependecies. Any suggestions?

Reply:
Ok this works too, since we are creating multiple instances of influxdatawriter, passing the config object would be quicker than loading the env variables each time